keps/sig-contributor-experience/1553-issue-triage/README.md

```diff
@@ -179,28 +168,27 @@ Upon determining that an issue is ready to be actively worked on, an org member
 can apply the `triage/accepted` label using the following bot command:
 
 ```shell
-/triage accept
+/triage accepted
 ```
 
 (Alternatively, contributors with write access to `kubernetes/kubernetes` would be
 able to manually apply the `triage/accepted` label.)
 
 **Considerations:**
 
-- `triage/accepted` should be considered a placeholder name for the
-  label in this provisional state, depending on what we decide is the most
-  appropriate name.
-- There will be an expectation that only SIG members designated to triage
-  issues are applying triage labels. We're considering limiting application of
-  this label to the [`milestone-maintainers` GitHub team](https://github.com/orgs/kubernetes/teams/milestone-maintainers).
+Any Org member may use the `/triage accepted` command in good faith that the
+issue is valid and belongs to the owning community group. Should it be widely
+misused, the command may be restricted to the [`milestone-maintainers`] or
+another to-be-determined GitHub team in the future.
```

Comment:
Can see a version of this workflow being very useful to SIG Docs.
I see the accepted status as a signal to people filing documentation bugs that their issue has been heard. Also it helps [PR wranglers](https://kubernetes.io/docs/contribute/participate/pr-wranglers/) filter on what needs attention.